### PR TITLE
replace GuzzleHttp\Psr7\build_query by GuzzleHttp\Psr7\Query:build

### PR DESCRIPTION
**Pull request by bastiendonjon**
PHP 8 and GUzzle >= 7.2 compatibility

### CODE REVIEW

**Comment by neoteknic** [comment]
+1 please fix

**Comment by SudoGetBeer** [comment]
@rajatsib could you merge this, please?

File: composer.json [hunk 1]
```diff
@@ -34,10 +34,5 @@
     },
     "autoload-dev": {
         "psr-4": { "SendinBlue\\Client\\" : "test/" }
-    },
```
Comment (rajatsib) [inline composer.json:37]: Hi @bastiendonjon 

Can you please retain this part?
It is needed

**Comment by rajatsib** [comment]
Added a Comment in the PR

File: composer.json (hunk 1 above)
Comment (bastiendonjon) [reply composer.json]: yes

**Comment by rajatsib** [comment]
Please use the latest version of PHP SDK (7.4.4)

**Pull request merged**